cruise-control/src/main/java/com/linkedin/kafka/cruisecontrol/common/Resource.java

```diff
@@ -18,7 +18,7 @@
   CPU("cpu", 0, true, true, 0.001),
   NW_IN("networkInbound", 1, true, false, 10),
   NW_OUT("networkOutbound", 2, true, false, 10),
-  DISK("disk", 3, false, true, 100);
+  DISK("disk", 3, false, true, 200);
```

Comment:
It is a little suspicious that the https://github.com/linkedin/cruise-control/issues/930 indicates a `0` value for sum of replica utilization. I am curious if there is a special reason for this case?

Reply:
The sanity check is on the cluster model after all goals are applied. In this case(broker removal), all the replicas are moved off the broker so the `0` value is accurate, the inaccurate values is actually the broker utilization...